server/selinux/server/pulp-streamer.te

```diff
@@ -76,3 +80,22 @@ ifdef(`distro_fedora23', `
     fs_rw_inherited_tmpfs_files(squid_t);
     fs_read_tmpfs_files(squid_t);
 ')
+
+
+##### Pulp stuff #####
+
+## Allow access to /var/run/pulp for EL 6
+ifdef(`distro_rhel6', `
+    allow streamer_t pulp_var_run_t:file manage_file_perms;
```

Comment:
Yeah I pretty much just stole it from the other one. Let's improve it later as you suggest.